PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+High level description including:
```

Comment:
This isn't a title or intended to be one. It's a "placeholder text you should replace with your own text in this formatting"